tasks/extract_filepaths_for_granule/test/unit_tests/test_extract_file_paths_for_granule.py

```diff
@@ -166,21 +166,21 @@ def test_task_no_filepath(self):
         self.task_input_event["config"]["private-bucket"] = "my_private_bucket"
         self.task_input_event["config"]["public-bucket"] = "my_public_bucket"
         self.task_input_event["config"]["file-buckets"] = [
-            {"regex": ".*.h5$", "sampleFileName": "L_10-420.h5", "bucket": "protected"},
+            {"regex": ".*.h5$", "sampleFileName": "L_10-420.h5", "bucket": {"name": "protected"}},
```

Comment:
Improper test here see comments on the main file and additional explanation below. 

In this context 
```
{
                "regex": ".*.iso.xml$",
                "sampleFileName": "L_10-420.iso.xml",
                "bucket": "protected",
}
```
The value of bucket points to the key in the buckets configuration. For example a bucket configuration like the following made during deployment may look like the following:

```
buckets = {
  default_orca = {
    name = "orca-sandbox-glacier"
    type = "orca"
  },
  internal = {
    name = "orca-bh-sandbox-internal"
    type = "internal"
  },
  private = {
    name = "orca-bh-sandbox-private"
    type = "private"
  },
  protected = {
    name = "orca-bh-sandbox-protected"
    type = "protected"
  },
  public = {
    name = "orca-bh-sandbox-public"
    type = "public"
  },
  provider = {
    name = "orca-sandbox-s3-provider"
    type = "provider"
  }
}
```
Per our example, the "protected" value in bucket would map to this object. Note the `protected` key
```
  protected = {
    name = "orca-bh-sandbox-protected"
    type = "protected"
  },
```
What this test should be validating is that we are properly mapping the regex to the proper s3 bucket name in the end. So per our example we would expect a dictionary entry like the following:
```
{ ".*.iso.xml$": "orca-bh-sandbox-protected"}